app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express")
+const app = express()
+const { getTopics } = require("./controllers/topics")
+const { getArticleById } = require('./controllers/article_id')
+
+app.get('/api/topics', getTopics)
+
+app.get('/api/articles/:article_id', getArticleById)
+
+app.get('/*', (req, res) => {
+    res.status(404).send({ message: 'Not Found' });
+  });
+
+  app.use((err, req, res, next) => {
+    console.error(err)
```

Comment:
Before merging your code to main, you must make sure that the only `console.log` in your code goes _inside_ the `500` error handling block _( the else block on line 18 )_